src/System.Collections/src/System.Collections.csproj

```diff
@@ -26,9 +28,15 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == 'net46'">
-    <TargetingPackReference Include="mscorlib" />
-    <TargetingPackReference Include="System" />
-    <TargetingPackReference Include="System.Core" />
+    <TargetingPackReference Include="mscorlib">
+      <InProject>false</InProject>
```

Comment:
This is a workaround for https://github.com/dotnet/corefx/issues/5042. Without this, basically you can't see any of the source files in VS, because the "System" TargetingPackReference node in the solution explorer hides the "System" folder with all of the source files under it. I'm okay with removing this for now, but we should think of some way to fix that.